schema-importer/app/src/test/kotlin/com/scalar/db/analytics/postgresql/schemaimporter/CreateSchemaTest.kt

```diff
@@ -49,8 +49,8 @@ class CreateSchemaTest {
         CreateSchema(ctx, namespaces).run()
 
         verify {
-            statement.executeUpdate("CREATE SCHEMA IF NOT EXISTS ns1;")
-            statement.executeUpdate("CREATE SCHEMA IF NOT EXISTS ns2;")
+            statement.executeUpdate("CREATE SCHEMA IF NOT EXISTS \"ns1\";")
```

Comment:
[trivial] Maybe we don't need the escape? But I'm not sure which is better.
```suggestion
            statement.executeUpdate("""CREATE SCHEMA IF NOT EXISTS "ns1";""")
```

Reply:
I see. I changed to use the triple double-quotes in https://github.com/scalar-labs/scalardb-analytics-postgresql/pull/45/commits/9f87de662447482935ff076f39acee5be7ea18ac.